src/utils/HypervBoxUtil.js

```diff
@@ -31,19 +28,19 @@ var HypervBox = {
   */
 
   hasAdminRights: function() {
-    return util.execFile([this.command(), 'Get-VMHostSupportedVersion']).then(stdout => {
```

Comment:
just found
```
PS C:\Users\Mario> get-command get-vmhost

CommandType     Name                                               Version    Source
-----------     ----                                               -------    ------
Cmdlet          Get-VMHost                                         2.0.0.0    Hyper-V


PS C:\Users\Mario> get-module -ListAvailable -Name Hyper-v


    Verzeichnis: C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules


ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Binary     2.0.0.0    Hyper-V                             {Add-VMAssignableDevice, Add-VMDvdDrive, Add-VMFibreChannelHba, Add-VMGroupMember...}
Binary     1.1        Hyper-V                             {Add-VMDvdDrive, Add-VMFibreChannelHba, Add-VMHardDiskDrive, Add-VMMigrationNetwork...}


PS C:\Users\Mario>
```


As you suggested. `FullyQualifiedErrorId` may be a good option